### PR TITLE
[STTNHUB-48] - Character count option for stories

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: assets/utils.js [hunk 1]
```diff
@@ -332,13 +332,50 @@ export function getTextFromHtml(html) {
     return text.join('');
 }
 
+
+/**
+ * Get label for the count field
+ */
+export function getCountLabel(titleCase) {
+    const text = gettext(getConfig('count_label', 'words'));
```
Comment (petrjasek) [inline assets/utils.js]: this won't extract the label for translation

**Comment by petrjasek** [comment]
btw according to the ticket they want to have both word and char count, not just one

**Comment by akintolga** [comment]
Hi @petrjasek you're right, I'll add character count as an extra. Thanks for the notification 👍

File: assets/wire/components/WireListItem.jsx [hunk 2]
```diff
@@ -98,7 +98,7 @@ class WireListItem extends React.Component {
                                 <div className='wire-articles__item__meta-info'>
                                     <span className='bold'>{this.slugline}</span>
                                     <span>{item.source}
-                                        {' // '}<span>{this.wordCount}</span> {gettext('words')}
+                                        {' // '}<span>{this.count}</span> {gettext('words')}
```
Comment (nrvikas) [inline assets/wire/components/WireListItem.jsx]: The variable (this.count) needs initialization (or assignment) - as this count value is not seen on the wire's list item.

**Pull request merged**